apis/Google.Cloud.Logging.V2/Google.Cloud.Logging.Log4Net/project.json

```diff
@@ -20,7 +20,8 @@
 
   "dependencies": {
     "Google.Cloud.Logging.V2": { "target": "project" },
-    "log4net": "2.0.5"
+    "log4net": "2.0.5",
+    "Google.Api.Gax": "1.0.0-beta08-CI00126" // Temp
```

Comment:
That's this // Temp comment about ?  If it's added manually, please either remove or make it complete.